registry/flutter_samples_gallery.test

```diff
@@ -0,0 +1,6 @@
+contact=raboughanem@google.com
+fetch=git -c core.longPaths=true clone https://github.com/flutter/samples.git tests
+fetch=git -c core.longPaths=true -C tests checkout 3a05f1f286c99a99a40c2035fd17488e9b654ba5
+fetch=git -c core.longPaths=true -C tests filter-branch --subdirectory-filter gallery
```

Comment:
What is this line for?

Reply:
This is required in order to run the test commands on the gallery subdirectory only